po/common/de.po

```diff
@@ -1,1295 +1,1219 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: \n"
+"Project-Id-Version: openfoodfacts\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2017-01-18 10:51+0000\n"
-"PO-Revision-Date: 2017-03-23 17:40+0100\n"
-"Last-Translator: Michael Glanznig <Unknown>\n"
-"Language-Team: \n"
+"PO-Revision-Date: 2017-04-08 13:41-0400\n"
+"Last-Translator: pierreslamich <pierre.slamich@gmail.com>\n"
+"Language-Team: German\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "X-Launchpad-Export-Date: 2017-01-18 10:56+0000\n"
-"X-Generator: Poedit 1.8.12\n"
-"Language: de\n"
+"X-Generator: crowdin.com\n"
+"Language: de_DE\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Crowdin-Project: openfoodfacts\n"
+"X-Crowdin-Language: de\n"
+"X-Crowdin-File: /master/po/common/en.po\n"
 
 #. put here the language tag ("fr" for French, "ja" for Japanese, etc)
 msgctxt ":langtag"
 msgid ":langtag"
-msgstr "de"
```

Comment:
yes, it's doing shit. The Merge Request was auto created. Please don't merge until I fix it.